concepts/numbers/introduction.md

```diff
@@ -26,9 +24,13 @@ You can see more details and discussions in the following resources:
 - [Documentation for `float()` built in][`float()` built in]
 - [0.30000000000000004.com][0.30000000000000004.com]
 
+### Precision
+
+Before diving into arithmetic, it is worth thinking about what precision means. Precision is the level of exactness that a number can be represented. An `int` is less precise than a `float` the same way that `1` is less precise that`1.125`.
+
 ## Arithmetic
 
-Python fully supports arithmetic between `ints` and `floats`. It will convert narrower numbers to match their less narrow counterparts when used with the binary arithmetic operators (`+`, `-`, `*`, `/`, `//`, and `%`). When division with `/`, `//` returns the quotient and `%` returns the remainder.
+Python fully supports arithmetic between `ints` and `floats`. It will convert narrower numbers to match their wider (or more precise) counterparts when used with the binary arithmetic operators (`+`, `-`, `*`, `/`, `//`, and `%`). When division with `/`, `//` returns the quotient and `%` returns the remainder.
```

Comment:
```suggestion
Python fully supports arithmetic between `ints` and `floats`. It will convert narrower numbers to match their wider (_or more precise_) counterparts when used with the binary arithmetic operators (`+`, `-`, `*`, `/`, `//`, and `%`). When division with `/`, `//` returns the quotient and `%` returns the remainder.
```